nautobot/wireless/api/serializers.py

```diff
@@ -0,0 +1,71 @@
+from rest_framework import serializers
+from rest_framework.validators import UniqueTogetherValidator
+
+from nautobot.core.api import ChoiceField, NautobotModelSerializer, ValidatedModelSerializer
+from nautobot.extras.api.mixins import (
+    TaggedModelSerializerMixin,
+)
+from nautobot.wireless import models
+from nautobot.wireless.choices import RadioProfileChannelWidthChoices
+
+
+class AccessPointGroupSerializer(TaggedModelSerializerMixin, NautobotModelSerializer):
+    class Meta:
+        model = models.AccessPointGroup
+        fields = "__all__"
+        validators = []
+
+    def validate(self, data):
+        if data.get("controller", None):
+            validator = UniqueTogetherValidator(
+                queryset=models.AccessPointGroup.objects.all(),
+                fields=["controller", "name"],
+            )
+            validator(data, self)
```

Comment:
Presuming this is the usual workaround for unique-together with a nullable field? Mind adding the usual documentation comment?

Reply:
Yep, I will add comments.